addons/project/views/project_views.xml

```diff
@@ -960,16 +981,31 @@
                         <page name="task_dependencies" string="Blocked By" attrs="{'invisible': [('allow_task_dependencies', '=', False)]}" groups="project.group_project_task_dependencies">
                             <field name="depend_on_ids" nolabel="1" context="{'default_project_id' : project_id}">
                                 <tree editable="bottom">
+                                    <field name="legend_normal" invisible="1"/>
+                                    <field name="legend_done" invisible="1"/>
+                                    <field name="legend_blocked" invisible="1"/>
+                                    <field name="message_needaction" invisible="1" readonly="1"/>
                                     <field name="is_closed" invisible="1" />
-                                    <field name="name" />
+                                    <field name="allow_subtasks" invisible="1" />
+                                    <field name="sequence" invisible="1" readonly="1"/>
+                                    <field name="priority" widget="priority" optional="show" nolabel="1"/>
+                                    <field name="name" widget="name_with_subtask_count"/>
+                                    <field name="child_text" invisible="1"/>
                                     <field name="project_id" optional="hide" />
+                                    <field name="partner_id" optional="hide"/>
+                                    <field name="parent_id" optional="hide" attrs="{'invisible': [('allow_subtasks', '=', False)]}" groups="base.group_no_one"/>
```

Comment:
Don't add this field because the parent one is the one displayed in the form view, so it is useless.

Reply:
it's the "blocked by" not "sub-tasks" so they may have different parents and belong to different projects. am I wrong ?

Reply:
Indeed, you can let it for the `depend_on_ids` field. Thanks